src/main/java/core/basesyntax/service/Parser.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.Operation;
+import java.util.List;
+
+@FunctionalInterface
```

Comment:
Why do you use this annotation here?

Reply:
to anounce that it contains 1 anonymous method